subprojects/glib.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+directory=glib
+url=https://gitlab.gnome.org/GNOME/glib.git
+push-url=git@gitlab.gnome.org:GNOME/glib.git
+revision=master
```

Comment:
I think we should bin this to a known good release.

Reply:
Pinned to latest stable release (2.58.1), and tested that version with MSVC.